website/src/main/java/com/google/step/data/CloudSQLManager.java

```diff
@@ -79,12 +79,21 @@ public void drop(String tableName) {
   //Get distinct specifcied columns from table matching given clauses if any
   public ResultSet getDistinct(String tableName, List<String> columns, List<String> clauses) throws SQLException {
     String values = String.join(", ", columns);
-    String where = (clauses == null) ? ";" : String.format(" WHERE %s;", String.join("AND", clauses)); 
+    String where = (clauses == null) ? ";" : String.format(" WHERE %s;", String.join(" OR ", clauses)); 
     String query = String.format("SELECT DISTINCT %s FROM %s%s", values, tableName, where);
     Statement stmt = this.conn.createStatement();
     return stmt.executeQuery(query);
   }
 
+  //Get all GN4P orgs similar to org
+  public ResultSet getPossibleComparisons(String tableName, OrganizationInfo org) throws SQLException {
+    List<String> attributes = Arrays.asList(
+        String.format("name LIKE %s", "'%" + org.getName() + "%'"),
+        String.format("link LIKE %s", "'%" + org.getLink() + "%'"),
```

Comment:
Optional: link likely will never be the same as a keyword, so I don't know how necessary this is, but ok to keep as it is.